src/SFML/Window/OSX/HIDInputManager.hpp

```diff
@@ -281,6 +281,7 @@ class HIDInputManager : NonCopyable
     // Member data
     ////////////////////////////////////////////////////////////
     IOHIDManagerRef    m_manager;                                             ///< Underlying HID Manager
+    bool               m_keysInitialized = false;                             ///< Has initializeKeyboard been called at least once?
```

Comment:
https://en.cppreference.com/w/cpp/language/data_members

NSDMI (yay for C++ initialisms) is a C++11 feature so we'll have to initialize this member in the construct initializer list instead.

Reply:
Oh shit! Had no idea... Okay, easy fix...